cmd/juju/status/formatter.go

```diff
@@ -210,6 +213,11 @@ func (sf *statusFormatter) formatApplication(name string, application params.App
 			meterStatuses:   application.MeterStatuses,
 		})
 	}
+
+	for bind, space := range application.EndpointBindings {
```

Comment:
This doesn't do anything except set the fields in the struct to the last value in the slice